Resources/Prototypes/_White/Actions/types.yml

```
@@ -0,0 +1,14 @@
+  name: Create hardlight spear
+  description: Creates hardlight spear in your hands.
+  noSpawn: true
+  components:
+  - type: InstantAction
+    useDelay: 1.5
+    itemIconStyle: BigAction
+    priority: -20
+    icon:
+      sprite: _White/Objects/Weapons/Melee/Spear/hardlight_spear.rsi
+      state: spear
+    event: !type:ActivateSpawnImplantEvent
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)